odl/__init__.py

```diff
@@ -43,7 +43,7 @@
 from . import deform
 from . import ufunc_ops
 from . import datasets
-
+from . import contrib
```

Comment:
I'm a bit confused now, the line `from . import contrib` is present in the odlgroup/odl/master version of  `odl/__init__.py ` (?)